src/crimson/os/seastore/lba_manager.h

```diff
@@ -111,7 +111,9 @@ class LBAManager {
     laddr_t addr) = 0;
 
   virtual void complete_transaction(
-    Transaction &t) = 0;
+    Transaction &t,
+    std::vector<CachedExtentRef> &,
+    std::vector<CachedExtentRef> &) = 0;
```

Comment:
Let's name these params and add a comment explaining what they are as in the rest of this file.